mdx-web/src/main/java/com/mx/web/mdx/models/Transfers/TransferRepaymentsQueryParameters.java

```diff
@@ -8,6 +8,5 @@ public class TransferRepaymentsQueryParameters {
 
   // NOTE: Non-standard naming to assist with parameter binding
   private String start_on;
-  private String transfer_type;
```

Comment:
Not being used anywhere and not in spec https://developer.internal.mx/drafts/mdx/transfer/index.html#transfers-transfer-repayment-schedule